doc/nrf/protocols/thread/certification.rst

```diff
@@ -177,3 +180,48 @@ This is valid only for Nordic Semiconductor development kits with a J-Link virtu
 
 To add an nRF52840 DK, drag the nRF52840 DK and drop it on the test bed configuration page.
 After that, the device is configured and the :ref:`proper baud rate (115200) <test_and_optimize>` and COM port are set.
+
+.. _ug_thread_build_report:
+
+OpenThread build report
+***********************
+
+The OpenThread build report contains information about the current |NCS| and OpenThread revisions, Thread feature set, Thread library path and changes in the :ref:`nrfxlib:nrfxlib` repository in comparison with the latest |NCS| release.
+The report is generated to the output console log, and stored as an additional build artefact in the application build directory.
+
+To enable building the Thread report set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT` kconfig option to ``y``.
+To disable generating the build artefact set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT` kconfig option to ``n``.
```

Comment:
```suggestion
You can disable generating the build artefact by setting the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT` kconfig option to ``n``.
```
Since this is more an option than something that needs to be done.